### ISSUE TITLE
ChangeList view not working

### ISSUE THREAD

**Issue by Louspirit** [ProjectBorealis/UE4GitPlugin#154]
The View Changelist doesn't show saved content not checked out.
![image](https://github.com/ProjectBorealis/UEGitPlugin/assets/2952456/68ffcbfa-8325-4e18-ac46-d0a3f51a1d69)

I see unsave modification though
![image](https://github.com/ProjectBorealis/UEGitPlugin/assets/2952456/5df7d9bf-2902-4da9-ba7f-bc48b42d3228)

But when I try to add it to a new changelist, I got this error
```
LogSlate: Window 'Move Files To...' being destroyed
SourceControl: Error: Operation 'NewChangelist' not supported by revision control provider 'Git LFS 2'
SourceControl: Error: Failed to move the file to the new changelist.
```
When I try to add it to an existing change list and then refresh, I get this error.
```
Message dialog closed, result: Ok, title: Message, text: Git command failed. Please check your connection and try again, or check the output log for more information.
LogSourceControl: Error: Command 'UpdateStatus' Failed!
LogAutomationController: Ignoring very large delta of 3.26 seconds in calls to FAutomationControllerManager::Tick() and not penalizing unresponsive tests 
```

**Comment by mastercoms** [comment]
It should be fixed with the latest version, let me know if you can

**Comment by Louspirit** [comment]
Edit: tested on UE5.3

I have the same behavior when trying to add to the new changelist. 

When trying to add to an existing one, I got this crash:

```
Fatal error: [File:D:\build\++UE5\Sync\Engine\Source\Runtime\CoreUObject\Private\UObject\UObjectGlobals.cpp] [Line: 373] Illegal call to StaticFindObjectFast() while serializing object data or garbage collecting!

UnrealEditor_CoreUObject!StaticFindObjectFast() [D:\build\++UE5\Sync\Engine\Source\Runtime\CoreUObject\Private\UObject\UObjectGlobals.cpp:377]
UnrealEditor_CoreUObject!StaticFindObject() [D:\build\++UE5\Sync\Engine\Source\Runtime\CoreUObject\Private\UObject\UObjectGlobals.cpp:504]
UnrealEditor_CoreUObject!FAssetData::GetClass() [D:\build\++UE5\Sync\Engine\Source\Runtime\CoreUObject\Private\AssetRegistry\AssetData.cpp:448]
UnrealEditor_SourceControlWindows!RefreshAssetInformationInternal() [D:\build\++UE5\Sync\Engine\Source\Editor\SourceControlWindows\Private\SSourceControlCommon.cpp:130]
UnrealEditor_SourceControlWindows!FOfflineFileTreeItem::RefreshAssetInformation() [D:\build\++UE5\Sync\Engine\Source\Editor\SourceControlWindows\Private\SSourceControlCommon.cpp:325]
UnrealEditor_SourceControlWindows!FOfflineFileTreeItem::FOfflineFileTreeItem() [D:\build\++UE5\Sync\Engine\Source\Editor\SourceControlWindows\Private\SSourceControlCommon.cpp:319]
UnrealEditor_SourceControlWindows!SSourceControlChangelistsWidget::OnRefreshUnsavedAssetsWidgets() [D:\build\++UE5\Sync\Engine\Source\Editor\SourceControlWindows\Private\SSourceControlChangelists.cpp:960]
UnrealEditor_SourceControlWindows!SSourceControlChangelistsWidget::OnRefreshUI() [D:\build\++UE5\Sync\Engine\Source\Editor\SourceControlWindows\Private\SSourceControlChangelists.cpp:1381]
UnrealEditor_SourceControlWindows!TBaseSPMethodDelegateInstance<0,SSourceControlChangelistsWidget,1,void __cdecl(FString const &),FDefaultDelegateUserPolicy>::ExecuteIfSafe() [D:\build\++UE5\Sync\Engine\Source\Runtime\Core\Public\Delegates\DelegateInstancesImpl.h:298]
UnrealEditor_UnsavedAssetsTracker!TMulticastDelegate<void __cdecl(FString const &),FDefaultDelegateUserPolicy>::Broadcast() [D:\build\++UE5\Sync\Engine\Source\Runtime\Core\Public\Delegates\DelegateSignatureImpl.inl:956]
UnrealEditor_UnsavedAssetsTracker!FUnsavedAssetsTracker::StopTrackingDirtyPackage() [D:\build\++UE5\Sync\Engine\Source\Developer\UnsavedAssetsTracker\Source\Private\UnsavedAssetsTracker.cpp:457]
UnrealEditor_UnsavedAssetsTracker!FUnsavedAssetsTracker::OnPackageDirtyStateUpdated() [D:\build\++UE5\Sync\Engine\Source\Developer\UnsavedAssetsTracker\Source\Private\UnsavedAssetsTracker.cpp:298]
UnrealEditor_UnsavedAssetsTracker!TBaseRawMethodDelegateInstance<0,FUnsavedAssetsTracker,void __cdecl(UPackage *),FDefaultDelegateUserPolicy>::ExecuteIfSafe() [D:\build\++UE5\Sync\Engine\Source\Runtime\Core\Public\Delegates\DelegateInstancesImpl.h:535]
UnrealEditor_CoreUObject!TMulticastDelegate<void __cdecl(UPackage *),FDefaultDelegateUserPolicy>::Broadcast() [D:\build\++UE5\Sync\Engine\Source\Runtime\Core\Public\Delegates\DelegateSignatureImpl.inl:956]
UnrealEditor_CoreUObject!UPackage::SetDirtyFlag() [D:\build\++UE5\Sync\Engine\Source\Runtime\CoreUObject\Private\UObject\Package.cpp:128]
UnrealEditor_CoreUObject!`anonymous namespace'::InnerSave() [D:\build\++UE5\Sync\Engine\Source\Runtime\CoreUObject\Private\UObject\SavePackage2.cpp:2951]
UnrealEditor_CoreUObject!UPackage::Save2() [D:\build\++UE5\Sync\Engine\Source\Runtime\CoreUObject\Private\UObject\SavePackage2.cpp:3083]
UnrealEditor_CoreUObject!UPackage::Save() [D:\build\++UE5\Sync\Engine\Source\Runtime\CoreUObject\Private\UObject\SavePackage.cpp:20]
UnrealEditor_UnrealEd!UEditorEngine::Save() [D:\build\++UE5\Sync\Engine\Source\Editor\UnrealEd\Private\EditorEngine.cpp:4707]
UnrealEditor_UnrealEd!UEditorEngine::SavePackage() [D:\build\++UE5\Sync\Engine\Source\Editor\UnrealEd\Private\EditorEngine.cpp:4716]
UnrealEditor_UnrealEd!UEditorEngine::Exec_Obj() [D:\build\++UE5\Sync\Engine\Source\Editor\UnrealEd\Private\EditorServer.cpp:4739]
UnrealEditor_UnrealEd!UEditorEngine::Exec_Editor() [D:\build\++UE5\Sync\Engine\Source\Editor\UnrealEd\Private\EditorServer.cpp:5810]
UnrealEditor_Core!FExec::Exec() [D:\build\++UE5\Sync\Engine\Source\Runtime\Core\Private\Misc\Exec.cpp:18]
UnrealEditor_Engine!UEngine::Exec() [D:\build\++UE5\Sync\Engine\Source\Runtime\Engine\Private\UnrealEngine.cpp:4686]
UnrealEditor_UnrealEd!UUnrealEdEngine::Exec() [D:\build\++UE5\Sync\Engine\Source\Editor\UnrealEd\Private\UnrealEdSrv.cpp:662]
UnrealEditor_UnrealEd!InternalSavePackage() [D:\build\++UE5\Sync\Engine\Source\Editor\UnrealEd\Private\FileHelpers.cpp:3528]
UnrealEditor_UnrealEd!InternalPromptForCheckoutAndSave() [D:\build\++UE5\Sync\Engine\Source\Editor\UnrealEd\Private\FileHelpers.cpp:4174]
UnrealEditor_UnrealEd!InternalCheckoutAndSavePackages() [D:\build\++UE5\Sync\Engine\Source\Editor\UnrealEd\Private\FileHelpers.cpp:5490]
UnrealEditor_UnrealEd!UEditorLoadingAndSavingUtils::SavePackages() [D:\build\++UE5\Sync\Engine\Source\Editor\UnrealEd\Private\FileHelpers.cpp:5536]
UnrealEditor_SourceControlWindows!`SSourceControlChangelistsWidget::OnOpenContextMenu'::`8'::<lambda_2>::operator()() [D:\build\++UE5\Sync\Engine\Source\Editor\SourceControlWindows\Private\SSourceControlChangelists.cpp:2940]
UnrealEditor_SourceControlWindows!TBaseFunctorDelegateInstance<void __cdecl(void),FDefaultDelegateUserPolicy,`SSourceControlChangelistsWidget::OnOpenContextMenu'::`8'::<lambda_2> >::ExecuteIfSafe() [D:\build\++UE5\Sync\Engine\Source\Runtime\Core\Public\Delegates\DelegateInstancesImpl.h:870]
UnrealEditor_Slate!TDelegate<void __cdecl(void),FDefaultDelegateUserPolicy>::ExecuteIfBound<void,0>() [D:\build\++UE5\Sync\Engine\Source\Runtime\Core\Public\Delegates\DelegateSignatureImpl.inl:570]
UnrealEditor_Slate!SMenuEntryBlock::OnClicked() [D:\build\++UE5\Sync\Engine\Source\Runtime\Slate\Private\Framework\MultiBox\SMenuEntryBlock.cpp:1167]
UnrealEditor_Slate!SMenuEntryBlock::OnMenuItemButtonClicked() [D:\build\++UE5\Sync\Engine\Source\Runtime\Slate\Private\Framework\MultiBox\SMenuEntryBlock.cpp:1118]
UnrealEditor_Slate!TBaseSPMethodDelegateInstance<0,SMenuEntryBlock,1,FReply __cdecl(void),FDefaultDelegateUserPolicy>::Execute() [D:\build\++UE5\Sync\Engine\Source\Runtime\Core\Public\Delegates\DelegateInstancesImpl.h:281]
UnrealEditor_Slate!TDelegate<FReply __cdecl(void),FDefaultDelegateUserPolicy>::Execute() [D:\build\++UE5\Sync\Engine\Source\Runtime\Core\Public\Delegates\DelegateSignatureImpl.inl:549]
UnrealEditor_Slate!SButton::ExecuteOnClick() [D:\build\++UE5\Sync\Engine\Source\Runtime\Slate\Private\Widgets\Input\SButton.cpp:465]
UnrealEditor_Slate!SButton::OnMouseButtonUp() [D:\build\++UE5\Sync\Engine\Source\Runtime\Slate\Private\Widgets\Input\SButton.cpp:390]
UnrealEditor_Slate!SMenuEntryButton::OnMouseButtonUp() [D:\build\++UE5\Sync\Engine\Source\Runtime\Slate\Private\Framework\MultiBox\SMenuEntryBlock.cpp:434]
UnrealEditor_Slate!FEventRouter::Route<FReply,FEventRouter::FToLeafmostPolicy,FPointerEvent,`FSlateApplication::RoutePointerUpEvent'::`8'::<lambda_3> >() [D:\build\++UE5\Sync\Engine\Source\Runtime\Slate\Private\Framework\Application\SlateApplication.cpp:442]
UnrealEditor_Slate!FSlateApplication::RoutePointerUpEvent() [D:\build\++UE5\Sync\Engine\Source\Runtime\Slate\Private\Framework\Application\SlateApplication.cpp:5206]
UnrealEditor_Slate!FSlateApplication::ProcessMouseButtonUpEvent() [D:\build\++UE5\Sync\Engine\Source\Runtime\Slate\Private\Framework\Application\SlateApplication.cpp:5775]
UnrealEditor_Slate!FSlateApplication::OnMouseUp() [D:\build\++UE5\Sync\Engine\Source\Runtime\Slate\Private\Framework\Application\SlateApplication.cpp:5740]
UnrealEditor_ApplicationCore!FWindowsApplication::ProcessDeferredMessage() [D:\build\++UE5\Sync\Engine\Source\Runtime\ApplicationCore\Private\Windows\WindowsApplication.cpp:2231]
UnrealEditor_ApplicationCore!FWindowsApplication::DeferMessage() [D:\build\++UE5\Sync\Engine\Source\Runtime\ApplicationCore\Private\Windows\WindowsApplication.cpp:2738]
UnrealEditor_ApplicationCore!FWindowsApplication::ProcessMessage() [D:\build\++UE5\Sync\Engine\Source\Runtime\ApplicationCore\Private\Windows\WindowsApplication.cpp:1099]
UnrealEditor_ApplicationCore!FWindowsApplication::AppWndProc() [D:\build\++UE5\Sync\Engine\Source\Runtime\ApplicationCore\Private\Windows\WindowsApplication.cpp:937]
user32
user32
UnrealEditor_ApplicationCore!FWindowsPlatformApplicationMisc::PumpMessages() [D:\build\++UE5\Sync\Engine\Source\Runtime\ApplicationCore\Private\Windows\WindowsPlatformApplicationMisc.cpp:148]
UnrealEditor!FEngineLoop::Tick() [D:\build\++UE5\Sync\Engine\Source\Runtime\Launch\Private\LaunchEngineLoop.cpp:5749]
UnrealEditor!GuardedMain() [D:\build\++UE5\Sync\Engine\Source\Runtime\Launch\Private\Launch.cpp:188]
UnrealEditor!GuardedMainWrapper() [D:\build\++UE5\Sync\Engine\Source\Runtime\Launch\Private\Windows\LaunchWindows.cpp:118]
UnrealEditor!LaunchWindowsStartup() [D:\build\++UE5\Sync\Engine\Source\Runtime\Launch\Private\Windows\LaunchWindows.cpp:258]
UnrealEditor!WinMain() [D:\build\++UE5\Sync\Engine\Source\Runtime\Launch\Private\Windows\LaunchWindows.cpp:298]
UnrealEditor!__scrt_common_main_seh() [D:\a\_work\1\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288]
kernel32
ntdll
````

**Comment by Louspirit** [comment]
Seems fixed in 5.4.4, thanks 😁
![image](https://github.com/user-attachments/assets/8a968ffd-a9ab-47bd-8dc7-3279fd250453)
